CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - After successful import of one or multiple bib entries the main table scrolls to the first imported entry [#5383](https://github.com/JabRef/jabref/issues/5383)
 - We fixed an exception which occurred when an invalid jstyle was loaded. [#5452](https://github.com/JabRef/jabref/issues/5452)
 - We fixed an error where the preview theme did not adapt to the "Dark" mode [#5463](https://github.com/JabRef/jabref/issues/5463)
+- We fixed an issue where the merge dialogue showed the wrong text colour in "Dark" mode [#5516](https://github.com/JabRef/jabref/issues/5516)
```

Comment:
Can you please write "dialog" instead of "dialogue"? Thanks!

Reply:
Fixed 😄